daemon/src/cfd_actors.rs

```diff
@@ -89,6 +90,21 @@ pub async fn load_cfd(order_id: OrderId, conn: &mut PoolConnection<Sqlite>) -> R
     Ok(cfd)
 }
 
+pub async fn apply_event(
```

Comment:
I don't think this abstraction is worth anything to be honest. We should be able to `??` this on all call-sites. Also, we could implement `xtra::Message` directly for `Event` then we wouldn't need to wrap it in `process_manager::Event` if you are concerned about boilerplate.

The error message can move into the message handler.

Reply:
so far it's saving me 6 lines of code on every callsite - I will clean this up as suggested in a follow-up refactor.

Reply:
> so far it's saving me 6 lines of code on every callsite

The way the function currently operates yes. But I'd argue it is wrong and we should fail instead of just logging! Generally, I am not a fan of these kind of helpers. I agree that duplicating these 6 lines is bad. In my experience though, hiding this pain will not actually get it solved because we forget about it and eventually, you end up with 7 layers of indirection until you reach the code that does what you want to do. Instead, I would like us to strive for making the actual code (sending the event to the process manager) as ergonomic as possible and until we are there, I am happy (and would perhaps even advocate for) leaving some pain behind to remind us of that :D